index.html

```diff
@@ -220,10 +228,7 @@ <h2 class="conta">CONTACT US</h2>
 <li class="M-icon">
 <i class="fa-solid fa-m"></i></li>
```

Comment:
- [ ]  As suggested by the previous reviewer, your social media links are meant to be clickable, so I kindly suggest that you wrap them in `a` tag 

## EXAMPLE
```
<li class="">
                  <a href="https://medium.com" target="_blank" rel="noopener noreferrer">
                    <img src="./images/svg/Github-icon.svg" alt="link to github" class="social_icon">
                  </a>
                </li>
```